pkg/maglev/maglev.go

```diff
@@ -21,6 +21,8 @@ import (
 const (
 	seed0 = 1836082274
 	seed1 = 1684172144
+
+	DefaultTableSize = 16381
```

Comment:
Given that we are multiplying by 1000, shouldn't this default be 67 so that MaxEntries would not be much bigger than 65536?